package/src/components/ImageGallery/components/ImageGalleryFooter.tsx

```diff
@@ -32,8 +31,6 @@ import {
 import { FileTypes } from '../../../types/types';
 import type { Photo } from '../ImageGallery';
 
-const SafeAreaView = SafeAreaViewOriginal ?? RNSafeAreaView;
```

Comment:
Hey @greenfrvr, everything looks fine but this this change is not needed as we decided to fall back to the default React Native's SafeAreaView just in case anyone doesn't it. As if we don't do it, it might be very much breaking for the integrators and not everyone is above 0.81.0 in RN as far as our integrators are concerned.

Reply:
Please add back the change everywhere else imo.

Reply:
returned it back. thanks!